templates/500.html

```diff
@@ -15,7 +15,7 @@ <h1>{% trans 'Maintenance ongoing' %}</h1>
             {% trans 'we are currently working on the Geoportal.' %} <br>
             {% trans 'The service will be back as fast as possible.' %}
             <hr>
-            {% trans 'Helpdesk contact:' %} +49 (0) 261/492-300 | helpdesk@vermkv.rlp.de
+            {% trans 'Helpdesk contact:' %} +49 (0) 611/535-5513 | verHVBG-Geoportal-Root@hvbg.hessen.de
```

Comment:
@Andreasr28 changed to this.

Reply:
Ok .. than this email pls : gdi-hessen@hvbg.hessen.de

Reply:
Not worth all the individual e-mail effort  .. Leave the root address in it